src/test/java/de/retest/recheck/printer/ActionReplayResultPrinterTest.java

```diff
@@ -104,4 +117,66 @@ void toString_should_print_message_if_no_recheck_action_replay_result() {
 
 		assertThat( string ).contains( NoGoldenMasterActionReplayResult.MSG_LONG );
 	}
+
+	@Test
+	void toString_should_not_print_child_differences_if_insertion_or_deletion() {
+		final List<ElementDifference> empty = Collections.emptyList();
+
+		final ElementDifference differences = change( Path.fromString( "html[1]/body[1]" ), Arrays.asList( //
+				delete( Path.fromString( "html[1]/body[1]/div[1]" ), Arrays.asList( // 
+						delete( Path.fromString( "html[1]/body[1]/div[1]/div[1]" ), Arrays.asList( // 
+								delete( Path.fromString( "html[1]/body[1]/div[1]/div[1]/div[1]" ), empty ), // 
+								delete( Path.fromString( "html[1]/body[1]/div[1]/div[1]/div[2]" ), empty ) //
+						) ), //
+						delete( Path.fromString( "html[1]/body[1]/div[1]/div[2]" ), empty ) //
+				) ), //
+				change( Path.fromString( "html[1]/body[1]/div[2]" ), Arrays.asList( // 
+						delete( Path.fromString( "html[1]/body[1]/div[2]/div[1]" ), Arrays.asList( // 
+								delete( Path.fromString( "html[1]/body[1]/div[2]/div[1]/div[1]" ), empty ), //
+								delete( Path.fromString( "html[1]/body[1]/div[2]/div[1]/div[2]" ), empty ) //
+						) ), //
+						change( Path.fromString( "html[1]/body[1]/div[2]/div[2]" ), empty ) ) //
+				) ) );
+
+		final RootElementDifference rootDifference =
+				new RootElementDifference( differences, mock( RootElement.class ), mock( RootElement.class ) );
+
+		final ActionReplayResult replayResult = ActionReplayResult.withDifference( ActionReplayData.ofSutStart(),
+				WindowRetriever.empty(), DifferenceRetriever.of( Collections.singletonList( rootDifference ) ), 0L );
+
+		ApprovalsUtil.verify( cut.toString( replayResult ) );
+	}
+
+	private ElementDifference delete( final Path path, final List<ElementDifference> childDifferences ) {
```

Comment:
Maybe signature of `delete` (and `change`) to accept both `( String path )` and `( String, List<ElementDifference> )`, then the actual test code is less verbose?